kpops/cli/main.py

```diff
@@ -372,17 +397,18 @@ def clean(
     dry_run: bool = DRY_RUN,
     verbose: bool = VERBOSE_OPTION,
 ):
-    pipeline_config = create_pipeline_config(config, defaults, verbose)
-    pipeline = setup_pipeline(
-        pipeline_base_dir, pipeline_path, components_module, pipeline_config
-    )
-    pipeline_steps = reverse_pipeline_steps(pipeline, steps, filter_type)
+    async def clean_runner(component: PipelineComponent):
+        await component.clean(dry_run)
```

Comment:
You should call destroy and then clean. Isn't that correct?
```suggestion
        await component.destroy(dry_run)
        await component.clean(dry_run)
```

Reply:
I probably messed with the order. Let me see again :)